scalableminds/webknossos
# Added more statistics to info tab

### Mailable description of changes:
- Tracing View: The info tab now includes the total number of trees, nodes, and branch points.

### Steps to test:
- Open skeleton tracing
- Check info tab to include the above mentioned infos
- Double check a volume tracing

### Issues:
- fixes #348 

------
